### ISSUE TITLE
Provide a mechanism to get a short list of random games

### ISSUE THREAD

**Issue by sgstair** [ludumdare/ludumdare#1194]
Ideally, weighted with the smart rating (where smart greater than zero)

So people can use this for on-the-spot random game selection for rating (I'd use the heck out of a feature like this, rather than purely browsing the smart ratings)

Efficient implementation may require caching smart rating + node id in an additional side table, in the magic task.

**Comment by povrazor** [comment]
Related to #954

**Comment by local-minimum** [comment]
I might look into making a widget for displaying games according to results from api call, basically a simplified feed view of the games view without options on how to filter results.

**Comment by povrazor** [comment]
The existing code that displays games can be improved for this purpose (that's how the "pinned" news works). 

On this subject, the box view needs improvement. Right now it's limited to 3 items wide inside the content view, and when a non-3 number of items is returned you get a jumbo item (due to there being no filler items). This needs to support way more choices (1 wide, 2 wide, 4 wide, 5 wide, etc).

**Comment by local-minimum** [comment]
Yeah I figured, doing the suggested widget would trigger reworking the existing. So many fun UI things to work on.
